src/lib/Server/Controller/Language.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformRest\Server\Controller;
+
+use eZ\Publish\API\Repository\LanguageService;
+use eZ\Publish\API\Repository\Values\Content\Language as ApiLanguage;
+use EzSystems\EzPlatformRest\Server\Controller as RestController;
+use EzSystems\EzPlatformRest\Server\Values\LanguageList;
+use Traversable;
+
+final class Language extends RestController
```

Comment:
legacy namespaces for new classes are technically not allowed after rebranding due to BC policy. I see 2 options here:
either use `Ibexa` namespace or make the class internal so we don't have to keep BC (we don't do it for controller classes anyway, but at least it would be more clear that way).

Reply:
@alongosz Replaced all new classes with new namespace in [a48f66f](https://github.com/ezsystems/ezplatform-rest/pull/107/commits/a48f66fa55ca5d95fa09336e295c9e7356fb36e8).